submissions/EmmanuelIdiaghe/database/transportDB.sql

```diff
@@ -13,7 +13,7 @@ create table passengers (
 	id int not null auto_increment,
     name_of_passenger varchar(100) not null,
     gender varchar(20) not null,
-    age int,
```

Comment:
why are you changing the data type to varchar?

Reply:
The Data Import Wizard skips all rows with incomplete data. For the passengers table, it imports only 714 rows instead of 891, skipping 177 passengers that have their 'age' column empty.

Changing the data type from int to varchar fixed the problem!

Reply:
Wow... that's weird. I didn't experience such an issue. Mine is `int` and my data where imported completely

Reply:
Yeah, I find it surprising too!

It could just be variation in versions of the workbench. Since the query does what it's supposed to do, should the data type of a column be much of a problem?

Reply:
ok. I think @Samju200 have the same issue of data not importing completely. You might want to reach out to him to find out how he was able to manage that issue. When you are done, reach out to me on your findings.